cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -251,6 +258,16 @@ func (scaleSet *ScaleSet) SetScaleSetSize(size int64) error {
 		Sku:      vmssInfo.Sku,
 		Location: vmssInfo.Location,
 	}
+
+	if vmssInfo.ExtendedLocation != nil {
```

Comment:
This (and ForceDelete) is from edge zone support https://github.com/kubernetes/autoscaler/pull/6447